### ISSUE TITLE
NGINX 1.5

### ISSUE THREAD

**Issue by splitice** [yaoweibin/nginx_syslog_patch#11]
src/core/ngx_log.c:699:27: error: ângx_log_tâ has no member named âfacilityâ
src/core/ngx_log.c:699:9: error: statement with no effect [-Werror=unused-value]
src/core/ngx_log.c:700:27: error: ângx_log_tâ has no member named âpriorityâ
src/core/ngx_log.c:700:9: error: statement with no effect [-Werror=unused-value]
src/core/ngx_log.c: In function âlog_exitâ:
src/core/ngx_log.c:721:23: error: ângx_log_tâ has no member named âsyslog_setâ
src/core/ngx_log.c: At top level:
src/core/ngx_log.c:674:1: error: ângx_set_syslogâ defined but not used [-Werror=unused-function]

etc

Attempted to use 1.4 patch.

**Comment by splitice** [comment]
Just so you know the hunks

Hunk #1 FAILED at 10.
Hunk #2 succeeded at 23 (offset 2 lines).
Hunk #3 succeeded at 54 (offset 2 lines).
Hunk #4 succeeded at 67 (offset 2 lines).
Hunk #5 FAILED at 144.
Hunk #6 succeeded at 156 (offset -1 lines).
Hunk #7 FAILED at 199.
Hunk #8 succeeded at 496 with fuzz 2 (offset 70 lines).
Hunk #9 succeeded at 545 with fuzz 2 (offset 13 lines).
Hunk #10 succeeded at 570 with fuzz 2 (offset 24 lines).
Hunk #11 succeeded at 664 with fuzz 2 (offset 60 lines).
3 out of 11 hunks FAILED -- saving rejects to file src/core/ngx_log.c.rej
patching file src/core/ngx_log.h
Hunk #2 FAILED at 68.
Hunk #3 FAILED at 228.
2 out of 3 hunks FAILED -- saving rejects to file src/core/ngx_log.h.rej
patching file src/http/modules/ngx_http_log_module.c

**Comment by msva** [comment]
[and, also, can you add new tag with patch for 1.5 then?]

**Comment by yaoweibin** [comment]
Thanks for your report. It seems nginx-1.5.2 change the error_log structure greatly. I'll try to fix this problem this weekend.

**Comment by msva** [comment]
Hi there!
Any news? :)

**Comment by splitice** [comment]
Yeah multiple error log directives are now supported, I can see how this might require more major changes. Take your time and get it right :)

**Comment by splitice** [comment]
Any progress towards 1.5.0?

**Comment by adnae** [comment]
Hi guys,

Any news about 1.5? :)

**Comment by splitice** [comment]
https://github.com/splitice/nginx_syslog_patch

Includes 1.5.6 support (access log only). I have no interest in doing error_log support at this time.

I upgraded it over the weekend, this patch hasnt had QA internally yet. Feel free to let me know if it has any problems.

**Comment by simao** [comment]
What would be needed to implement error_log support?

**Comment by moll** [comment]
Switching to the latest official version of Nginx where this is already built-in might do it. ;-)

**Comment by simao** [comment]
I tried that. it doesn't work properly. I am currently submitting some bugs to the nginx project, like this https://bugs.launchpad.net/nginx/+bug/1329400

Also has another bug when syslogd is restart nginx doesn't properly reopen the socket so all the log msgs are lost.
